ci/matrix.yaml

```diff
@@ -25,33 +24,33 @@ workflows:
     - {jobs: ['test_lid1',  'test_lid2'], project: ['cub'], std: 'max',    cxx: ['gcc']}
     - {jobs: ['test_nolid', 'test_lid0'], project: ['cub'], std: 'max',    cxx: ['clang', 'msvc']}
     # Modded builds:
-    - {jobs: ['build'], std: 'all', ctk: '12.5', cxx: 'nvhpc'}
-    - {jobs: ['build'], std: 'all', cxx: ['gcc', 'clang'], cpu: 'arm64'}
-    - {jobs: ['build'], std: 'all', cxx: ['gcc'], sm: '90a'}
+    - {jobs: ['build'], std: 'minmax', ctk: '12.5', cxx: 'nvhpc'}
+    - {jobs: ['build'], std: 'minmax', cxx: ['gcc', 'clang'], cpu: 'arm64'}
+    - {jobs: ['build'], std: 'minmax', cxx: ['gcc'], sm: '90a'}
```

Comment:
Testing 17 and 20 is far more important to me on the latest version of compilers than testing 11 and 20.